llm_toolkit/models.py

```diff
@@ -263,6 +264,67 @@ class GPT4o(GPT):
   name = 'gpt-4o'
 
 
+class Claude(LLM):
+  """Anthropic's Claude model encapsulator."""
+
+  _max_output_tokens = 4096
+  context_window = 200000
+
+  name = "claude-3-haiku@20240307"
```

Comment:
nit: should we hide the version number from the actual users of oss-fuzz-gen? 

This is how we handle Gemini for example: https://github.com/google/oss-fuzz-gen/blob/bdcc09b3ed2c8b85929ce262009847b436a0ba5f/llm_toolkit/models.py#L461

We have a public "vertex_ai_gemini-1-5" name that can be passed to the CLI, but under the hood "gemini-1.5-pro-001" is what gets passed to the Vertex AI API.

Reply:
Done